src/rocprof_compute_soc/soc_base.py

```diff
@@ -393,26 +416,8 @@ def perfmon_coalesce(pmc_files_list, perfmon_config, workload_dir):
 
         pmc = []
         for block_name in f.blocks.keys():
-            if block_name == "TCC":
```

Comment:
Does this apply to v2 still?

Reply:
I believe so. The unit tests are all running with v2 and they work fine